### PR TITLE
Bash 03 hw task 2 find dup files

### PR DESCRIPTION
**Pull request by ghostzero2000**
Implement find_dup_files.sh

### CODE REVIEW

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+
+current_dir="$(pwd)"
+
+function view_help {
+	echo "Script find_dup_files.sh usage information."
+	echo "SYNOPSIS"
+	echo "find_dup_files.sh [DIR] [-h]"
+	echo -e "Display information about files with identical md5, if the directory is not specified, then the current directory is by default.\n"
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh]: Too long line

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		*) 
+		wrong_option
+		;;
+	esac
+
+fi
+
+find_similar_files $current_dir
+
+
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh:51]: 2 empty lines in the end of file

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 1 above)
Comment (ghostzero2000) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Ok. Thank you. I am changing it.

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 2 above)
Comment (ghostzero2000) [reply 03-Bash/find_dup_files/find_dup_files.sh:51]: Ok. I am removing it.

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 1 above)
Comment (yekovalyov) [reply 03-Bash/find_dup_files/find_dup_files.sh]: I don't see any change in this file. Please check the updated branch.

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 2 above)
Comment (yekovalyov) [reply 03-Bash/find_dup_files/find_dup_files.sh:51]: I don't see any change in this file. Please check the updated branch.

Comment (ghostzero2000) [reply 03-Bash/find_dup_files/find_dup_files.sh:51]: Hello. I updated the branch.

File: 03-Bash/find_dup_files/find_dup_files.sh (hunk 1 above)
Comment (ghostzero2000) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Hello. I updated the branch.

File: 03-Bash/find_dup_files/find_dup_files.sh [hunk 3]
```diff
@@ -7,7 +7,7 @@ function view_help {
 	echo "Script find_dup_files.sh usage information."
 	echo "SYNOPSIS"
 	echo "find_dup_files.sh [DIR] [-h]"
-	echo -e "Display information about files with identical md5, if the directory is not specified, then the current directory is by default.\n"
+	echo -e "Display information about files with identical md5,\n if the directory is not specified, then the current\n directory is by default.\n"
```
Comment (yekovalyov) [inline 03-Bash/find_dup_files/find_dup_files.sh]: Please check your commit using checkpatch.pl and fix warnings.

Comment (ghostzero2000) [reply 03-Bash/find_dup_files/find_dup_files.sh]: Ok. Thank you. I fixed it used /scripts/checkpatch.pl -g HEAD, linux version 4.14 and updated the branch

**Comment by ghostzero2000** [comment]
I updated the branch

**Pull request merged**